### DISCUSSION TITLE
Antd Body Row Memo Cause Memory Leaks in Fast and Dynamic Data

### DISCUSSION THREAD

**Discussion by silviaphungky** [ant-design/ant-design#45762] [Q&A]
**context:**
I used antd table (version ^5.8.1) and I have really dynamic and fast data from websocket.
I saw in the react component antd table is memoized the table row. I don't want that behaviour because I think that cause the garbage collection is failed so it become memory leaks. 
Here is the screenshoot of the antd table row:
<img width="162" alt="Screenshot 2023-11-09 at 15 50 22" src="https://github.com/ant-design/ant-design/assets/55534481/da7460ed-53ee-4c86-b4fe-7ad7e85dc00f">

The question is **how can I disabled the memo in antd table?** Thankyou

**Comment by zombieJ** [comment]
React do memo compare is for its props. Whatever using `React.memo` or not, the props of current fiber node is always in the memory.

You can modify `node_modules` to remove it to check if its related. But I do believe it will help nothing.

**Reply by silviaphungky** [reply]
if you see in version 4, there is no **Memo** tag in the **BodyRow**

<img width="1436" alt="Screenshot 2023-11-09 at 18 07 45" src="https://github.com/ant-design/ant-design/assets/55534481/ac818ff8-5de6-4a3a-b6ce-246e48e1fb80">

and this in version 5 @zombieJ 


<img width="1185" alt="Screenshot 2023-11-09 at 18 12 38" src="https://github.com/ant-design/ant-design/assets/55534481/215ad228-24d4-46b5-809e-fd39f72a05bf">

**Reply by silviaphungky** [reply]
more detail about the problem is I integrate running trade with antd table version 5 and it cause my memory become larger (up to 1GB), then I tried to debug the problem until I found there was a memoized value that failed to garbage collection. I already checked all of the components and functions, there is no react memo that I used to show the running trade until I found the memo BodyRow in antd table version 5.
when I change to pure HTML table, the memory usage reduce
